sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -345,4 +345,57 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_accelerometer</EFDB_Topic>
+    <Description>LabJack connected accelerometer unit. Unit is identified by sensorName. Raw data are published, 200 values in each burst (so 200 values @ 1 Hz are for 200 Hz sampling).</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The user-defined name of the accelerometer (e.g. AuxTel_accel1).</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Starting time of data acquisition (TAI unix seconds).</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>interval</EFDB_Name>
+      <Description>Data collection time interval. N-th (starting from 0, 0 indexed) sample is assumed to be taken at "timestamp" + ("interval"/200) * N.</Description>
```

Comment:
I gently suggest deleting ", 0 indexed" because "starting from 0" is already quite clear (and I found saying it twice made it more confusing).

That is a minor picky point; this is an excellent description.